cpu/rpx0xx/include/periph_cpu.h

```diff
@@ -400,6 +400,29 @@ typedef struct {
     IRQn_Type irqn;     /**< IRQ number of the UART interface */
 } uart_conf_t;
 
+/**
+ * @brief   Prevent shared timer functions from being used
+ */
+#define PERIPH_TIMER_PROVIDES_SET
+
+/**
+ * @brief   Configuration type of a timer channel
+ */
+typedef struct {
+    IRQn_Type irqn;                 /**< timer channel interrupt number */
+} timer_channel_conf_t;
+
+/**
+ * @brief   Configuration type of a timer device @ref timer_conf_t::dev,
+ *          having @ref timer_conf_t::chn number of channels,
+ *          each one modeled as @ref timer_channel_conf_t
+ */
+typedef struct {
+    TIMER_Type *dev;                /**< pointer to timer base address */
+    const timer_channel_conf_t *ch; /**< pointer to timer channel configuration */
+    uint8_t chn;                    /**< number of timer channels */
```

Comment:
```suggestion
    uint8_t ch_numof;               /**< number of timer channels */
```

The postfix `numof` is commonly used within RIOT's code base, maybe use it here as well for consistency?